openpype/plugins/publish/collect_anatomy_context_data.py

```diff
@@ -42,7 +42,7 @@ class CollectAnatomyContextData(pyblish.api.ContextPlugin):
     }
     """
 
-    order = pyblish.api.CollectorOrder + 0.002
+    order = pyblish.api.CollectorOrder - 0.02
```

Comment:
Change of this order makes this PR review complixity +1000 levels

Reply:
Fair, I can remove the ordering from the PR if it still works for you without the order change, I will keep it in my fork. I don't quite get why collector plugins that can run almost independently aren't set to the earliest possible order...

Reply:
does it make sense that for a much more complex collect like `CollectSettingsSimpleInstances` it's set to `- 0.49` but for one as easy as context data is `+ 0.002`?